### PR TITLE
Disable chronyd when using ptp

### PR DESCRIPTION
**Pull request by SchSeba**
This commit add a machine config to disable chronyd under the ptp kustomize

Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

**Comment by SchSeba** [comment]
@vitus133 please take a look

**Comment by vitus133** [comment]
@SchSeba 
Thanks, got it.

File: feature-configs/cn-ran-overlays/ran-profile/cluster-config/du-common/ptp/disable-chronyd.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            Documentation=man:chronyd(8) man:chrony.conf(5)
+            After=ntpdate.service sntp.service ntpd.service
+            Conflicts=ntpd.service systemd-timesyncd.service
+            ConditionCapability=CAP_SYS_TIME
+
+            [Service]
+            Type=forking
+            PIDFile=/run/chrony/chronyd.pid
+            EnvironmentFile=-/etc/sysconfig/chronyd
+            ExecStart=/usr/sbin/chronyd $OPTIONS
```
Comment (fedepaol) [inline feature-configs/cn-ran-overlays/ran-profile/cluster-config/du-common/ptp/disable-chronyd.yaml:26]: Is $OPTIONS being set somewhere?

Comment (SchSeba) [reply feature-configs/cn-ran-overlays/ran-profile/cluster-config/du-common/ptp/disable-chronyd.yaml:26]: nope this is the file that exist in RHCOS I just copy the file here the important part is the `enable:false` all the other is just to don't change the file content

Comment (fedepaol) [reply feature-configs/cn-ran-overlays/ran-profile/cluster-config/du-common/ptp/disable-chronyd.yaml:26]: understood, thanks

**Comment by fedepaol** [comment]
@SchSeba mind adding some context on why we are disabling chronyd?

Also, would it make sense to have it part of the ptp deployment instad of the profile / the demo layer?
Which boils down to: does it make sense to expect this to be deployed and should the validation step check for it?

**Comment by SchSeba** [comment]
I put it here because I don't want to ptp deployment to reboot the node (when we apply the MC) also if the user deploy ptp it may take time until he apply the policy and the node may lose time sync.

**Pull request merged**